programmers/카펫.js

```diff
@@ -0,0 +1,16 @@
+function solution(brown, yellow) {
+    const result = [];
+    let length  = 0;
+    
+    for (let i = 1; i < yellow + 1; i++) {
```

Comment:
```suggestion
    for (let i = 1; i < yellow + 1; i++) {
```
for문에서 i는 단순히 반복을 하기위한 임의의 값이 아니라 yellow의 세로를 의미하기 때문에 변수 이름을 width, height 등으로 해주는게 가독성이 더 좋을 것 같아요.

Reply:
for문은 i j k를 쓰는게 일반적인걸로 알고 있는데 말씀하신 부분은 of 나 in에 혹시 해당되는게 아닌가요? for문을 이름을 바꾼는건 처음들어서요 ㅎㅎ

Reply:
예전에  멘토분한테 리뷰받았을때도  for문 변수을 i jk 안쓰고  바꿔서 지적받은적이 있거든요

Reply:
흠 만약 그런게 문제가 된다면 변수를 하나 지정해서 해줘야 겠네욥

Reply:
i값을  반복 용도로만 사용안하고 , 세로를 의미하려면
let h = i; 로 해서 사용해도될것같아요